CMakeLists.txt

```diff
@@ -334,6 +334,18 @@ if (USE_OPENSSL)
     message(FATAL_ERROR "TimescaleDB requires OpenSSL version 1.0 or greater")
   endif ()
 
+  if(MSVC)
```

Comment:
Shouldn't this happen only for debug builds?

Reply:
Not critical since the release build will not have libraries that need to be fixed, but added a check for it anyway.